### ISSUE TITLE
Error in margin calculation in State Audit Report

### ISSUE THREAD

**Issue by sfsinger19103** [FreeAndFair/ColoradoRLA#755]
Note 255 - 60 <> 150, so why is the "Margin" for Dan Chapin 150 when he got 255 votes and his nearest competitor got 60?
[state-recall-2017-11-06-report-2017-09-21T11_33_14.xlsx](https://github.com/FreeAndFair/ColoradoRLA/files/1322359/state-recall-2017-11-06-report-2017-09-21T11_33_14.xlsx)

Choice | W/L | Votes | Margin | Diluted Margin %
-- | -- | -- | -- | --
Dan Chapin | W | 225 | 150 | 13.1579
Michael Bennet | L | 60 | 0 | 0.0000
Darryl Glenn | L | 60 | 0 | 0.0000
Paul Noel Fiorino | L | 60 | 15 | 1.3158
Lily Tang Williams | L | 45 | 15 | 1.3158
Arn Menconi | L | 30 | 0 | 0.0000
Bill Hammons | L | 30 | 30 | 2.6316
Don Willoughby | L | 0 |   |

**Comment by dmzimmerman** [comment]
It would help to know how you generated this table so I can test it with the changes I’m making today to reporting.

**Comment by dmzimmerman** [comment]
Or, instead, perhaps you could try to replicate your results on the branch in https://github.com/FreeAndFair/ColoradoRLA/pull/752 (or on `master`, if it's been merged in already).

**Comment by sfsinger19103** [comment]
Logged in as countyadmin3. Uploaded the CVR_Export_1140.csv file (redacted) and its associated ballot manifest. 

State chose US Senate contest. 

Arapahoe Audit Board hit "ballot card not found" on all ballot cards in Round 1.

Then pulled the report. 

Let me know if you need copies of the BM/CVR files.

**Comment by dmzimmerman** [comment]
OK... the contents of the report (in terms of vote totals) don't depend on what happens in the audit, at all, so I'll try it first just using that CVR file with the smoketest and see what I get.

**Comment by dmzimmerman** [comment]
I see a margin of 165 when testing with the smoketest. Do you still see a margin of 150 when you run the interactive test?

**Comment by sfsinger19103** [comment]
OK, trying to recreate. After State launches audit but before any action by County I pulled the report and the margin was correct (165).

Set Risk Limit 30 (different from first time when I saw the issue)

Went through the 16 ballots in the round (hit ballot not found for all), pulled report and margin was correct.

State started Round Two, pulled report again, margin ws correct again.

Can't replicate it, so closing issue.
